File: Tasks/Java Work/SumOfEvenAndOddNumbers.java
import java.util.Scanner;
	public class SumOfEvenAndOddNumbers{
		public static void main(String[] args){
		Scanner input = new Scanner(System.in);
System.out.println("How many integers do you want to input?");
int integers = input.nextInt();

int sumOfEven = 0;
int sumOfOdd = 0;
for(int counter = 1; counter <= integers; counter ++){
System.out.println("Enter a number:");
int number = input.nextInt();

if (number % 2 == 0)
sumOfEven += number;

if (number % 2 != 0)
sumOfOdd += number;
}

System.out.println("The sum of even numbers is " + sumOfEven);
System.out.println("The sum of odd numbers is " + sumOfOdd);
}
}

File: Tasks/Python Work/sum_of_even_and_odd_numbers.py
integers = int(input("How many integers do you want to input? "))

sum_of_even = 0
sum_of_odd = 0
for counter in range(integers):
	number = int(input("Enter a number: "))
	if number % 2 == 0:
		sum_of_even = sum_of_even + number

	if number % 2 != 0:
		sum_of_odd = sum_of_odd + number

print("The sum of even numbers =", sum_of_even)
print("The sum of odd numbers =", sum_of_odd)
